src/ucp/rndv/proto_rndv.c

```diff
@@ -717,13 +637,26 @@ ucp_proto_rndv_bulk_init(const ucp_proto_multi_init_params_t *init_params,
     mpriv->align_thresh = ucs_max(rndv_align_thresh,
                                   mpriv->align_thresh + mpriv->min_frag);
 
-    /* Add ack latency */
-    status = ucp_proto_rndv_ack_init(&init_params->super.super, ack_name,
-                                     &multi_caps,
-                                     ucs_linear_func_make(150e-9, 0),
-                                     &rpriv->super, caps);
-    ucp_proto_select_caps_cleanup(&multi_caps);
+    status = ucp_proto_rndv_ack_init(&init_params->super, ack_name,
+                                     ucs_linear_func_make(50e-9, 0),
```

Comment:
overhead changed from 150e-9 to 50e-9, is that intentional?

Reply:
Yes, that helped to fix a couple of performance issues.